lib/parallel_tests/tasks.rb

```diff
@@ -102,7 +102,11 @@ def parse_args(args)
   # just load the schema (good for integration server <-> no development db)
   desc "load dumped schema for test databases via db:schema:load --> parallel:load_schema[num_cpus]"
   task :load_schema, :count do |t,args|
-    command = "rake db:schema:load RAILS_ENV=#{ParallelTests::Tasks.rails_env}"
+    if Gem::Version.new(Rails.version) > Gem::Version.new('4.2.0')
+      command = "rake db:purge db:schema:load RAILS_ENV=#{ParallelTests::Tasks.rails_env}"
```

Comment:
As I mentioned in the other PR [this commit](https://github.com/rails/rails/commit/5762a71685e178c19c60494fe8fe3a2ed2016742) rails will no longer clear out the database. In the case that your test suite or setup seeds data, it will cause the application to raise an error about a Foreign Key Constraint. Arguably I could remove the else portion of the if statement, but I saw it and the other PR and thought if that was helping others then I would include it.